NuRadioMC/SignalProp/analyticraytracing.py

```diff
@@ -1545,7 +1558,10 @@ class describing the index-of-refraction profile
         # make sure that arrays are floats
         self.__logger = logging.getLogger('ray_tracing')
         self.__logger.setLevel(log_level)
+        
         self.__medium = medium
+        self.__birefringence_model = birefringence_model
```

Comment:
To me it looks weird that you would have an ice model and a birefringence model. I think the information should be contained in 1 object (a medium which is or is not a birefringence model) because otherwise you could mix up the greenland model with a southpole birefringence model.

Reply:
I reworked the structure of the ice model such that the birefringence ice model can be used as a regular medium in the ray_tracer class. No need for a second ice model. Let me know if the overall structure of the model is what you had in mind.